My Series List/My Series List/View/MainView/MainViewPresenter.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Presenter.swift
+//  My Series List
+//
+//  Created by Ivan Dias on 24/11/2021.
+//
+
+import Foundation
+
+protocol MainViewDelegate: NSObjectProtocol {
+    
+    func startLoading()
+    func finishLoading()
+    func listSeries(_ series: [CoreDataSeries])
+    func listEmpty()
+}
+
+class MainViewPresenter {
```

Comment:
Rename as MainPresenter or ListPresenter. I would prefer a non-ambiguous name as List instead of Main that doesn't really specify what this screen does